site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -522,9 +522,10 @@ function printPreview(target){
 		<table>
 		#if ($providedGroups.size() > 0)
 			#foreach ($group in $providedGroups)
-				#if (!$group.name.contains("(suspended)"))
-					<tr>
-						<td>$validator.escapeHtml($group.name)</td>
+				<tr>
+					<td>$validator.escapeHtml($group.name)</td>
+##					don't show the Remove button if it's suspended group - it gets removed by removing the corrsponding non-suspended group
+					#if (!$group.name.contains("(suspended)"))
```

Comment:
There's group id and group name on the page. We can't really use group Id which is something like '2c918084530e065101530e0d38410006,2c918084530e065101530e0d384b0007' and group name can be 'DPhil Ancient History (Full-time), Current students (access)' or 	
'DPhil Ancient History (Full-time), Suspended students (suspended)' which has the role at the end.  I don't think we're likely to change the role name in the future (without a lot to think about) so it shouldn't be too brittle.